jetty-core/jetty-demos/jetty-demo-handler/src/main/config/modules/demo-handler.mod

```diff
@@ -0,0 +1,21 @@
+# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+
+[description]
+Demo Core Handler
+
+[tags]
+demo
+handler
+
+[depends]
+server
+
+[files]
+basehome:modules/demo.d/demo-handler.xml|etc/demo-handler.xml
+maven://org.eclipse.jetty.demos/jetty-demo-handler/${jetty.version}/jar|lib/jetty-demo-handler.jar
```

Comment:
Is this correct that it ends in `/jar`, before the `|`?

Reply:
yes. the format is `maven://<groupId>/<artifactId>/<version>/<type>/<classifier>`
with `type` optional default to `jar`